.github/workflows/performance.yml

```diff
@@ -0,0 +1,79 @@
+name: "Compare performance"
+
+on:
+  workflow_call:
+
+env:
+  GIT_CONFIG_GLOBAL: "/root/.gitconfig" # fix path in container (https://github.com/actions/runner/issues/2033)
+
+jobs:
+  compare-performance:
+    if: ${{ github.event_name == 'pull_request' }}
+    runs-on: "ubuntu-latest"
+    container:
+      image: "ghcr.io/tarantool/sdvg-ci:0.0.2"
+    permissions:
+      pull-requests: write
+      packages: read
+
+    steps:
+      - uses: "actions/checkout@v4"
+
+      - name: "Install requirements"
+        run: "make bench/prepare"
+
+      - name: "Try fetch cached benchmark result from master branch"
+        id: "cache-benchmark"
+        uses: "actions/cache/restore@v4"
+        with:
+          key: "benchmark-master"
+          path: "benchmark-master.txt"
+
+      - name: "Fetch master branch"
```

Comment:
I suggest simplifying the workflow and, if there are no metrics from the master branch, simply write the current metrics without deltas